src/components/Connect/Burner/CreateSeed/index.tsx

```diff
@@ -73,78 +70,62 @@ const Create: FC<{
-        <LoadingScreen />
-      ) : (
-        <form
-          onSubmit={e => {
-            e.preventDefault()
-            nextStep()
-          }}
-        >
-          <Box
-            display='flex'
-            flexDirection='column'
-            justifyContent='center'
-            alignItems='center'
+  return loading || walkthroughStep === 4 ? (
+    <LoadingScreen />
+  ) : (
+    <WideDialog>
+      <form
+        onSubmit={e => {
+          e.preventDefault()
+          nextStep()
+        }}
+      >
+        <ShadowBox>
+          <StepHeader currentStep={walkthroughStep} totalSteps={3} />
+          {mnemonic && (
+            <Walkthrough
+              importSeedError={importSeedError}
+              canContinue={canContinue}
+              walkthroughStep={walkthroughStep}
+              mnemonic={mnemonic}
+              setCanContinue={setCanContinue}
+            />
+          )}
+        </ShadowBox>
+        <ButtonRowSpaced>
+          <ButtonV2
+            large
+            type='button'
+            onClick={() =>
+              walkthroughStep === 1
+                ? back()
+                : setWalkthroughStep(walkthroughStep - 1)
+            }
           >
-            <OnboardCard
-              display='flex'
-              flexDirection='row'
-              flexWrap='wrap'
-              justifyContent='center'
-              maxWidth={16}
-              border={0}
-              css={`
-                background-color: var(--blue-gray);
-              `}
-            >
-              <StepHeader currentStep={walkthroughStep} totalSteps={3} />
-              {mnemonic && (
-                <Walkthrough
-                  importSeedError={importSeedError}
-                  canContinue={canContinue}
-                  walkthroughStep={walkthroughStep}
-                  mnemonic={mnemonic}
-                  setCanContinue={setCanContinue}
-                />
-              )}
-            </OnboardCard>
-            <ButtonRowSpaced>
-              <ButtonV2
-                type='button'
-                onClick={() => {
-                  if (walkthroughStep === 1) back()
-                  else if (walkthroughStep === 2) setWalkthroughStep(1)
-                }}
-              >
-                Back
-              </ButtonV2>
-              {walkthroughStep === 1 ? (
-                <ButtonV2 type='submit' large green>
-                  I&apos;ve recorded my seed phrase
-                </ButtonV2>
-              ) : (
-                <ButtonV2 type='submit' large green>
-                  Next
-                </ButtonV2>
-              )}
-            </ButtonRowSpaced>
-          </Box>
-        </form>
-      )}
-    </>
+            Back
+          </ButtonV2>
+          <ButtonV2 large green type='submit'>
+            {walkthroughStep === 1 ? "I've recorded my seed phrase" : 'Next'}
+          </ButtonV2>
+        </ButtonRowSpaced>
+      </form>
+    </WideDialog>
```

Comment:
I think the main takeaways here are that `OnBoardCard` could be deprecated in favor of `ShadowBox`. Also the custom styled `Box` wrapper wasn't necessary here. I added a `WideDialog` component that is `50em` instead of `35em` which can be used here.